boards/arm/mimxrt1062_fmurt6/doc/index.rst

```diff
@@ -0,0 +1,402 @@
+these references:
+
+- `i.MX RT1060 Website`_
+- `i.MX RT1060 Reference Manual`_
+- `MIMXRT1062-FMURT6 User Guide`_
+- `MIMXRT1062-FMURT6 Schematics`_
+
+Supported Features
+==================
+
+The mimxrt1062_fmurt6 board configuration supports the hardware features listed
+below.  For additional features not yet supported, please also refer to the
+:ref:`mimxrt1064_evk` , which is the superset board in NXP's i.MX RT10xx family.
+NXP prioritizes enabling the superset board with NXP's Full Platform Support for
+Zephyr.  Therefore, the mimxrt1064_evk board may have additional features
+already supported, which can also be re-used on this mimxrt1060_evk board:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| DISPLAY   | on-chip    | display                             |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | QSPI flash                          |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| SDHC      | on-chip    | disk access                         |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| ENET      | on-chip    | ethernet                            |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+| CAN       | on-chip    | can                                 |
++-----------+------------+-------------------------------------+
+| DMA       | on-chip    | dma                                 |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc                                 |
++-----------+------------+-------------------------------------+
+| SAI       | on-chip    | i2s                                 |
```

Comment:
Have all these IP blocks been enabled and tested with this board?

Reply:
I have not tested them, as it is not yet required. I can remove them from the table if you suggest.

Reply:
Understood. Please do add any IP blocks that have been tested and verified with this board to this list, I just want to make sure that we don't confuse users if an IP block has not been tested on this board.

Reply:
Added all the relevant IP blocks that have been tested and removed the non tested ones

Reply:
@danieldegrasse .. This section and the connections and IO section has been appropriately updated now